### DISCUSSION TITLE
Can't use ctrl-p and ctrl-n to move across groups

### DISCUSSION THREAD

**Discussion by capfredf** [emacs-helm/helm#2627] [Q&A] [answered]
I just updated my Helm to 20231108.1729, but I find myself unable to use ctrl-p/n (or left/right) to move the cursor across different sources. I am not sure if I should file a bug or this is just related to a config option.

**Comment by ghost** [comment, marked as answer]
Fred Fu ***@***.***> writes:

> I just updated my Helm to 20231108.1729, but I find myself unable to
> use ctrl-p/n (or left/right) to move the cursor across different
> sources. I am not sure if I should file a bug or this is just related
> to a config option.

AFAIK the latter: helm-move-to-line-cycle-in-source.


Michael.

**Comment by thierryvolpiatto** [comment]
Michael Heerdegen ***@***.***> writes:

> 1.  ( ) text/plain          (*) text/html           
>
> Fred Fu ***@***.***> writes:
>
>> I just updated my Helm to 20231108.1729, but I find myself unable to
>> use ctrl-p/n (or left/right) to move the cursor across different
>> sources. I am not sure if I should file a bug or this is just related
>> to a config option.
>
> AFAIK the latter: helm-move-to-line-cycle-in-source.

Yes For C-n/p but this doesn't affect left/right (next/previous source).

> Michael.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.*Message ID: ***@***.***>

-- 
Thierry
